bower.json

```diff
@@ -10,5 +18,8 @@
         "node_modules",
         "resources",
         "tasks"
+    ],
+    "dependencies": [
+        "jquery": ">=2.0.0"
```

Comment:
@minddust : please update this line if necessary to indicate correct version, or ask me to do here. Also if necessary, we should add a dependency line for Bootstrap as well.